configs/e2e.conf

```diff
@@ -3,6 +3,8 @@
 --nosimulator_obstacle_detection
 --obstacle_detection_model_paths=dependencies/models/obstacle_detection/faster-rcnn/
 --obstacle_detection_model_names=faster-rcnn
+--path_coco_labels=dependencies/models/pylot.names
```

Comment:
This flag already existed, but added it here explicitly as a reminder that we may need to change this depending on the model that we are using & the class labels that are supported.